models/Models.js

```diff
@@ -18,4 +18,69 @@ exports.getAllTopics = () => {
          
             return result.rows[0];
           });
-        }
\ No newline at end of file
+        }
+
+    // exports.getAllArticles =()=>{
+        
+    //     return db
+    //     .query(`SELECT * FROM articles;`)
+    //     .then((result) => {
+    //         const articlesArray = [];
+    //         result.rows.forEach(article => {
+    //             const articleObject = {
+    //                 author: article.author,
+    //                 title: article.title,
+    //                 article_id: article.article_id,
+    //                 topic: article.topic,
+    //                 created_at: article.created_at,
+    //                 votes: article.votes,
+    //                 article_img_url: article.article_img_url
+    //             }
+    //             const commentCount = `SELECT COUNT(*) AS comment_count FROM comments WHERE article_id = $1;`;
+    //             return db
+    //                 .query(commentCount,[article.article_id])
+    //                 .then((result) => {
+    //                     articleObject.comment_count = result.rows[0].comment_count
+    //                     articlesArray.push(articleObject)
+    //                 })
+                    
+    //         });
+    //     return articlesArray;
+    //     })
+    // }
+    exports.getAllArticles = () => {
+        return db
+        .query('SELECT * FROM articles;')
```

Comment:
This model could be quite a lot simpler. It would be fine to add the comment count part of the query in the line above